mne_bids/dig.py

```diff
@@ -281,24 +280,24 @@ def _write_coordsystem_json(*, raw, unit, hpi_coord_system,
         Defaults to False.
 
     """
-    dig = raw.info['dig']
-    if dig is None:
-        dig = []
-
-    coord_frame = set([dig[ii]['coord_frame'] for ii in range(len(dig))])
-    if len(coord_frame) > 1:  # noqa E501
-        raise ValueError('All HPI, electrodes, and fiducials must be in the '
-                         'same coordinate frame. Found: "{}"'
-                         .format(coord_frame))
+    if raw.get_montage() is None or raw.get_montage().get_positions() is None:
```

Comment:
Why would montage not be None, but positions are None?

Reply:
I got an error in the tests that positions were none I thought, let me double check

Reply:
Okay I might be wrong. Maybe it's a downstream consequence of mne